File: book/petrol-pump/pump/Sale.java
package pump;

public class Sale
{
    public Sale(Fuel fuel, double price, double cost, double quantity) {
        this.fuel = fuel;
        this.price = price;
        this.cost = cost;
        this.quantity = quantity;
    }

    public final Fuel fuel;
    public final double price;
    public final double cost;
    public final double quantity;
}

File: c#/Book/PetrolPump/PetrolPump/PetrolPump/Pump/Sale.cs
namespace PetrolPump.Pump
{
  public class Sale
  {
    public Sale(Fuel fuel, double price, double cost, double quantity) {
        Fuel = fuel;
        Price = price;
        Cost = cost;
        Quantity = quantity;
    }

    public readonly Fuel Fuel;
    public readonly double Price;
    public readonly double Cost;
    public readonly double Quantity;

  }
}
